geopandas/io/sql.py

```diff
@@ -7,60 +7,27 @@
 from geopandas import GeoDataFrame
 
 
-def read_postgis(
-    sql,
-    con,
-    geom_col="geom",
-    crs=None,
-    index_col=None,
-    coerce_float=True,
-    parse_dates=None,
-    params=None,
-):
+def _df_to_geodf(df, geom_col="geom", crs=None):
     """
-    Returns a GeoDataFrame corresponding to the result of the query
-    string, which must contain a geometry column in WKB representation.
+    Transfoms a pandas DataFrame into a GeoDataFrame.
+    The column geo_col must be a geometry column in WKB representation.
```

Comment:
A little more description about this being intended for dataframes generated from pandas sql reads of geospatial databases would probably be helpful. Also, "transfoms" -> "transforms" and "geo_col" -> "geom_col"